errors.go

```diff
@@ -92,18 +104,19 @@ func (err *HTTPError) Cause() error {
 	return err.wrapped
 }
 
-// translateError coerces err into an HTTPError that can be marshaled directly
+// translateError coerces err into an httpErrorResponse that can be marshaled directly
```

Comment:
If you make it public:
```suggestion
// translateError coerces err into an HTTPErrorResponse that can be marshaled directly
```